modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/view/DeployedService.java

```diff
@@ -40,6 +42,20 @@ public class DeployedService {
     @Schema(description = "The catalog of the service")
     private Category category;
 
+    /**
+     * The region of the Service.
+     */
+    @NotNull
+    @Schema(description = "The region of the provider.")
```

Comment:
It better to update the description `The region of the service.`

Reply:
fixed